src/components/ConferenceItem/ConferenceItem.js

```diff
@@ -38,20 +38,30 @@ export default class ConferenceItem extends PureComponent {
             {name}
           </Link>
         </Heading>
-        <p className={styles.p}>
-          {`${Location(city, country)}・`}
-          <span className={styles.Date}>
-            {formatDate(startDate, endDate)}
-          </span>
-        </p>
-        <p className={classNames(styles.p, styles.Footer)}>
-          {showCFP && <Cfp url={cfpUrl || url} date={cfpEndDate} />}
-          {showCFP && <br />}
-          <Topics topics={topics} />
-          {twitter && ' – '}
-          <Twitter twitter={twitter} />
-        </p>
-      </div>
+        <dl className={styles.dl}>
+          <div>
+            <dt className="visuallyHidden">Location</dt>
+            <dd>
+              {`${Location(city, country)}`}
+            </dd>
+          </div>
+          <div>
+            <dt className="visuallyHidden">Date</dt>
+            <dd>
+              <span aria-hidden="true">・</span>{formatDate(startDate, endDate)}
+            </dd>
+          </div>
+          <div className={classNames(styles.cfp, styles.Footer)}>
+            {showCFP && <Cfp url={cfpUrl || url} date={cfpEndDate} />}
```

Comment:
You can do:
```
{showCFP && 
  <div className={classNames(styles.cfp, styles.Footer)}>
    <Cfp url={cfpUrl || url} date={cfpEndDate} />
  </div>
}
```
to prevent an empty div to be rendered.